C/primitive/elements/primitive.h

```diff
@@ -182,6 +182,20 @@ typedef struct transaction {
   uint_fast32_t lockTime;
 } transaction;
 
+/* A structure representing taproot spending data from an Elements transaction.
+ * 'annexHash' is a cache of the hash of the annex.
+ *
+ * Invariant: branchLen <= 128
+ *            sha256_midstate branch[branchLen];
+ */
+typedef struct tapEnv {
+  const sha256_midstate *annexHash;
+  const sha256_midstate *branch;
+  sha256_midstate internalKey;
+  unsigned char branchLen;
+  unsigned char leafVersion;
+} tapEnv;
```

Comment:
This loses the information about output key parity (`controlblock[0] & 0x01`). Is there any special reason to not support it.

Reply:
There is technically no loss of information because the output key parity is a function of the internal pubkey and the merkle root, which is derivable from the rest of the control block.

That said, if we decide later to add an `outputKey` primitive, which I'm somewhat inclined to do, it probably makes sense to add this parity bit to it there.